proposed/0000-safecoin-market-price.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: SafeCoin_Market_Price
+- Type: New feature
+- Related components: maidsafe_vault, maidsafe_client, maidsafe_types
+- Start Date: 19-07-2015
+- RFC PR: (leave this empty)
+- Issue number: (leave this empty)
+
+# Summary
+
+This RFC is based on the concerns I’ve read in the MaidSafe forum for the last six months. These are some of them:
+
+* https://forum.safenetwork.io/t/lowest-put-price-discovery-and-new-safecoin-issuance/3819
+* https://forum.safenetwork.io/t/cost-of-storing-data-on-safe/4285
+* https://forum.safenetwork.io/t/the-price-of-safecoin-the-economics-behind-it/844
+
+This RFC is only another point of view (maybe not doable, feel free to reject this RFC) that breaks a lot the main design. Maybe this RFC can be decomposed in several RFC’s, smaller and feasible.
+
+
+# Motivation
+
+My first concern is how much cost to store a chunk of data (cost in terms of USD,EUR or BTC). No one is able to answer this question accurately. I’m thinking in a true Offer/Demand market, where you establish how much you want to pay to store data, and in the other side, the vaults establish how much they want to get paid to store that data.
+
+Another important aspect is the idea of pay once and store data forever. I can’t see how can be sustainable in the future. I’d like to see a system where you Put data into the Net as a renting service, like the Amazon Web Services.
+
```

Comment:
Yes that's correct, otherwise the network would have to recognise all data (which is a major security issue) There is an RFC  coming for deletable data, this will be much more expensive than normal de-duplicate data though.